test/integration/model/nested_include/base/base.sdf

```diff
@@ -0,0 +1,158 @@
+<?xml version="1.0" ?>
+<sdf version="1.7">
+  <model name="base">
+    <pose>0 0 0.325 0 -0 0</pose>
+
+    <link name='chassis'>
+      <pose>-0.151427 -0 0.175 0 -0 0</pose>
+      <inertial>
+        <mass>1.14395</mass>
+        <inertia>
+          <ixx>0.126164</ixx>
+          <ixy>0</ixy>
+          <ixz>0</ixz>
+          <iyy>0.416519</iyy>
+          <iyz>0</iyz>
+          <izz>0.481014</izz>
+        </inertia>
+      </inertial>
+      <visual name='visual'>
+        <geometry>
+          <box>
+            <size>2.01142 1 0.568726</size>
+          </box>
+        </geometry>
+        <material>
+          <ambient>0.5 0.5 1.0 1</ambient>
+          <diffuse>0.5 0.5 1.0 1</diffuse>
+          <specular>0.0 0.0 1.0 1</specular>
+        </material>
+      </visual>
+      <collision name='collision'>
+        <geometry>
+          <box>
+            <size>2.01142 1 0.568726</size>
+          </box>
+        </geometry>
+      </collision>
+
+      <visual name="lidar_visual">
+        <pose>-0.8 0 0.41 0 0 0</pose>
+          <geometry>
+              <cylinder>
+                  <radius>0.1</radius>
+                  <length>0.25</length>
+              </cylinder>
+          </geometry>
+      </visual>
+
+      <sensor name='lidar' type='gpu_lidar'>
+        <pose>-0.8 0 0.41 0 0 0</pose>
+        <topic>lidar</topic>
+        <update_rate>10</update_rate>
+        <lidar>
+          <scan>
+            <horizontal>
+              <samples>640</samples>
+              <resolution>1</resolution>
+              <min_angle>-1.396263</min_angle>
+              <max_angle>1.396263</max_angle>
+            </horizontal>
+            <vertical>
+              <samples>16</samples>
+              <resolution>1</resolution>
+              <min_angle>-0.261799</min_angle>
+              <max_angle>0.261799</max_angle>
+            </vertical>
+          </scan>
+          <range>
+            <min>0.08</min>
+            <max>10.0</max>
+            <resolution>0.01</resolution>
+          </range>
+        </lidar>
+        <alwaysOn>1</alwaysOn>
+        <visualize>true</visualize>
+      </sensor>
+
+      <sensor name="camera" type="camera">
+        <pose>-1.06 0 0 0 0 3.14</pose>
+        <camera>
+          <horizontal_fov>1.047</horizontal_fov>
+          <image>
+            <width>320</width>
+            <height>240</height>
+          </image>
+          <clip>
+            <near>0.1</near>
+            <far>100</far>
+          </clip>
+        </camera>
+        <always_on>1</always_on>
+        <update_rate>30</update_rate>
+        <visualize>true</visualize>
+        <topic>chassis/camera</topic>
+      </sensor>
+
+    </link>
+
+    <include>
+      <uri>top</uri>
```

Comment:
Can you set the name to `top_model` here just to make it clear that the model from uri `top` is called `top_model`.

Reply:
e8919a107af56891d1acf75ec61bcd646fdf0a7c